netket/optimizer/qgt/qgt_jacobian.py

```diff
@@ -38,7 +51,7 @@ class JacobianSMatrix(AbstractSMatrix):
     the field `sr`.
     """
 
-    O: jnp.ndarray
+    O: jnp.ndarray = Uninitialized
```

Comment:
Why is this better than making `O` a required argument?

Reply:
Because there is a default argument before (in the base class), `diag_shift`...
This must have a default value too.

Reply:
It's shit. It's been fixed in python 3.10, I should investigate if it's possible to back port the fix

Reply:
I see :facepalm: